src/stan_math_backend/Transform_Mir.ml

```diff
@@ -1,6 +1,49 @@
 open Core_kernel
 open Middle
 
+let opencl_triggers =
+  String.Map.of_alist_exn
+    [ ("normal_id_glm_lpdf", [0; 1])
+    ; ("bernoulli_logit_glm_lpmf", [0; 1])
+    ; ("categorical_logit_glm_lpmf", [0; 1])
+    ; ("neg_binomial_2_log_glm_lpmf", [0; 1])
+    ; ("ordered_logistic_glm_lpmf", [0; 1])
+    ; ("poisson_log_glm_lpmf", [0; 1]) ]
```

Comment:
@t4c1 All these now require both matrix_cl arguments to be data and already on the GPU. Do you think we could be less conservative in any of them? Just double checking.

Reply:
It is in most cases enough to have X already on GPU.